src/client/components/AddNewRoom/RoomHandler.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+
+import AddRoom from './AddRoom';
+
+const RoomHandler = () => {
```

Comment:
I think you wanted to put this into containers instead of components. It should be appropriately renamed.